src/assertions/text.js

```diff
@@ -0,0 +1,6 @@
+import { boolAssertBuilder } from './assertion-builder';
+
+boolAssertBuilder('containsText',
```

Comment:
[See comment](https://github.com/rkotze/should-enzyme/pull/5#issuecomment-252748805)